index.html

```diff
@@ -72,5 +88,183 @@
+				<div class="tag">bootstrap</div>
+				<div class="tag">ruby</div>
+			</div>
+			<div class="see-2">See Project</div>
+		</div>
+
+		<!-- Start of Cards -->
+		<div class="cards">
+			<p class="work-sample"> Profesional Art Printing Data </p>
+			<p class="work-sample-text">
+				A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+			</p>
+			<div class="tags">
+				<div class="tag">html</div>
+				<div class="tag">bootstrap</div>
+				<div class="tag">ruby</div>
+			</div>
+			<div class="see">See Project</div>
+		</div>
+		<div class="cards">
+			<p class="work-sample"> Profesional Art Printing Data </p>
+			<p class="work-sample-text">
+				A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+			</p>
+			<div class="tags">
+				<div class="tag">html</div>
+				<div class="tag">bootstrap</div>
+				<div class="tag">ruby</div>
+			</div>
+			<div class="see">See Project</div>
+		</div>
+		<div class="cards">
+			<p class="work-sample"> Profesional Art Printing Data </p>
+			<p class="work-sample-text">
+				A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+			</p>
+			<div class="tags">
+				<div class="tag">html</div>
+				<div class="tag">bootstrap</div>
+				<div class="tag">ruby</div>
+			</div>
+			<div class="see">See Project</div>
+		</div>
+		<div class="cards">
+			<p class="work-sample"> Profesional Art Printing Data </p>
+			<p class="work-sample-text">
+				A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+			</p>
+			<div class="tags">
+				<div class="tag">html</div>
+				<div class="tag">bootstrap</div>
+				<div class="tag">ruby</div>
+			</div>
+			<div class="see">See Project</div>
+		</div>
+		<div class="cards">
+			<p class="work-sample"> Profesional Art Printing Data </p>
+			<p class="work-sample-text">
+				A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+			</p>
+			<div class="tags">
+				<div class="tag">html</div>
+				<div class="tag">bootstrap</div>
+				<div class="tag">ruby</div>
+			</div>
+			<div class="see">See Project</div>
+		</div>
+		<div class="cards">
+			<p class="work-sample"> Profesional Art Printing Data </p>
+			<p class="work-sample-text">
+				A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+			</p>
+			<div class="tags">
+				<div class="tag">html</div>
+				<div class="tag">bootstrap</div>
+				<div class="tag">ruby</div>
+			</div>
+			<div class="see">See Project</div>
+		</div>
+	</div>
+</section>
+<br> <br> <br>
+<section class="head">
+	<!-- Heading Section  -->
+	<div class="flex-about">
+	  <div class="about-me">
+	  	<div class="head-title"> 
+	    	About Me 
+	    </div>
```

Comment:
- [ ] Kindly be noticed that the title (e.g., "About myself") uses a header tag `hx`.